.github/workflows/r_valgrind.yml

```
@@ -0,0 +1,29 @@
+name: R valgrind tests
+
+on:
+  pull_request_review_comment:
+    types: [created]
+
+jobs:
+  test-r-valgrind:
+    name: r-package (ubuntu-latest, R-devel, valgrind)
+    if: github.event.comment.body == '/gha run r-valgrind' && contains('OWNER,MEMBER,COLLABORATOR', github.event.comment.author_association)
+    timeout-minutes: 120
+    runs-on: ubuntu-latest
+    container:
+      image: wch1/r-debug
```

Should work according to the docs:
 > When you only specify a container image, you can omit the `image` keyword.
  https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer
```suggestion
    container: wch1/r-debug
```

---

BTW, the same can be done with
https://github.com/microsoft/LightGBM/blob/332559c0f84c2a23ec01c07737814b0aeb0ce474/.github/workflows/r_package.yml#L166-L167

---

oooo ok! I'll change the UBSAN here too

---

I decided to fix the UBSAN one in #3468 so it isn't held up by this PR